### PR TITLE
FE PROJECT CONFIG

### PR DESCRIPTION
**Pull request by Marek-Kazimierczak**
Add project configuration:

- Prettier
- Eslint
- Husky
- Lint-staged

### CODE REVIEW

**Comment by Marek-Kazimierczak** [comment]
I had a trouble to apply `"prettier --config .prettierrc --write"` in lint-staged so it is removed.

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "linebreak-style": ["error", "unix"],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+    "react/prop-types": 0
```
Comment (MarcinCholewka) [inline frontend/.eslintrc.json:25]: add few more rules :
- proptypes’y w componentach alfabetycznie
```javascript
    "react/sort-prop-types": [2, {
      "callbacksLast": false,
      "ignoreCase": true,
      "requiredFirst": false,
      "sortShapeProp": true
    }],
```
- require trailing commas on multilines
`'comma-dangle': [2, 'always-multiline'],`
- show if debugger was left
`'no-debugger': 2,`
- Require === and !== 
`'eqeqeq': 2,`
- enforce the consistent use of either double or single quotes in JSX attributes
`"jsx-quotes": [2, "prefer-single"],`
- enforce consistent spacing before and after keywords
`'keyword-spacing': 2,`

Comment (MarcinCholewka) [reply frontend/.eslintrc.json:25]: after call don't use
- enforce the consistent use of either double or single quotes in JSX attributes
"jsx-quotes": [2, "prefer-single"],

**Comment by MarcinCholewka** [comment]
I checked only the config files. I didn't check changes made by this config.

**Pull request merged**